docs/build/limits.md

```diff
@@ -2,14 +2,30 @@
 title: Limits
 ---
 
-OpenFn has a number of configurable limits that help ensure smooth operation.
-This page details what those limits are and how they can be adjusted.
+OpenFn cloud hosted instance has a number of limits that help ensure smooth
+operation. This page details what those limits are and how they can be adjusted.
```

Comment:
@christad92 maybe we link to openfn.org/pricing page to say "see here for the latest plans" in case we open one page but not the other?

Reply:
Right.. will update.